### PR TITLE
Custom commands grouping docs

### PR DESCRIPTION
**Pull request by sean-e-dietrich**
Updated Custom Commands doc to include the recent #526 Pull Request.

### CODE REVIEW

**Comment by achekulaev** [comment]
Pls review from your side @lmakarov

File: docs/fin/custom-commands.md [hunk 1]
```diff
@@ -163,6 +163,61 @@ services:
       - VIRTUAL_HOST
 ```
 
+## Grouping Commands
+
+In certain cases you may want to group your commands so that you can commit them to a repository and easily share them with others, similar to a toolkit of commands that
+can be shared across a team of developers. Docksal allows for the ability to group these commands within a folder and are usable.
```
Comment (lmakarov) [inline docs/fin/custom-commands.md]: >Docksal allows for the ability to group these commands within a folder and are usable.

Something is missing in this sentence.

File: docs/fin/custom-commands.md [hunk 2]
```diff
@@ -163,6 +163,61 @@ services:
...
+
+When running `fin help` there should be similar output like below. This will show you the available commands.
+
+```
+Docksal control cli utility v1.54.4
```
Comment (lmakarov) [inline docs/fin/custom-commands.md]: No need to include the whole `fin help` output, just the header and the custom commands section.

**Comment by sean-e-dietrich** [comment]
Thanks @lmakarov I realized my sentence was a little weird.

**Comment by lmakarov** [comment]
Merged. Thanks!

**Pull request merged**